tracer/src/Datadog.Trace/Iast/VulnerabilityTypeName.cs

```diff
@@ -43,6 +58,13 @@ public static string GetName(VulnerabilityType vulnerability)
             VulnerabilityType.WeakHash => WeakHash,
             VulnerabilityType.SqlInjection => SqlInjection,
             VulnerabilityType.CommandInjection => CommandInjection,
+            VulnerabilityType.PathTraversal => PathTraversal,
```

Comment:
Not for this PR, but FYI you could use our `[EnumExtensions]` attribute to trigger the source generator that can take care of this for you instead of maintaining it by hand. It supports using the `[Description]` attribute to override the default names:

```csharp
internal enum VulnerabilityType
{
    [Description("WEAK_CIPHER")]
    WeakCipher,
...
```